apps/testingaccessibility/src/components/team/index.tsx

```diff
@@ -68,11 +67,10 @@ const InviteTeam: React.FC<React.PropsWithChildren<InviteTeamProps>> = ({
       )}
       {!redemptionsLeft && (
         <div className="flex items-center justify-between mt-5 pt-5 border-t border-gray-100">
-          <Link href="/#buy">
-            <a className="font-semibold bg-green-500 transition text-white px-4 py-2 hover:bg-green-600 rounded-md flex-shrink-0">
-              Buy more seats
-            </a>
-          </Link>
+          <p className="py-3">
+            You are out of seats. To invite more people to your team, buy more
+            seats.
+          </p>
```

Comment:
This 'Buy more seats' link doesn't make sense anymore now that the `<BuyMoreSeats />` component is going to display right below this component on the Team Invite page.